api/v2alpha1/go/pipelinespec/pipeline_spec.pb.go

```diff
@@ -7,6 +7,7 @@
 package pipelinespec
 
 import (
+	status "google.golang.org/genproto/googleapis/rpc/status"
```

Comment:
revert this file?

Reply:
Shall we update this file to register the new changes in pipelineSpec? Maybe we can do it in another PR.